tests/unit/validation/test_validation.py

```diff
@@ -838,7 +838,8 @@ def test_validate_jacoco_partials(self):
                 {
                     "comment": {
                         "require_bundle_changes": True,
-                        "bundle_change_threshold": ("absolute", 1200),
+                        # "bundle_change_threshold": ("absolute", 1200),
```

Comment:
Hmm, can we remove the commented code, or link the ticket to clean up here? Commented out code (especially without context) can get confusing/stale pretty quick.